### ISSUE TITLE
Create modal to add money

### ISSUE THREAD

**Issue by maaxow** [maaxow/myCashWallet#17]
Create new modal to add money. Allow multiple adding !
Like this model :
![cw add](https://user-images.githubusercontent.com/6973084/31086446-208820e8-a79a-11e7-8d51-34db46d7f5c9.png)
When click on "+" => add new row. 
When click on "ADD" => add all rows in mongo.

**Comment by ypahalajani** [comment]
@maaxow : I can contribute to this. I am starting to work on this.

**Comment by maaxow** [comment]
No problem. Thank you very much @ypahalajani 😄

**Comment by ypahalajani** [comment]
@maaxow : i am unable to bootstrap this app. Getting too many errors. FIxed some
1. angular app.init() was not being able to execute because the module was being referenced in an incorrect way.
2. addController was added as a dependency but not defined => Ok, I had to write logic for this, So, for the time being, I commented it in dependency injection
3. Corrected the import of 'cw.app'.

following errors still persists: not able to instantiate 'cw.controller'. Need help?

PS: i was not familiar with the amd modules using CommonJS wrapper. Now, i have some idea of how modules are defined in amd pattern and accessed.

**Comment by maaxow** [comment]
@ypahalajani : thanks for contributing.

I make a lot of corrections on my branch. Do you want i merge with master, i think their change can help you a lot.

And yes, my bad, i make a lot of mistake with the previous commit...

**Comment by maaxow** [comment]
@ypahalajani  I thinks it's good 😄

**Comment by ypahalajani** [comment]
@maaxow : yeah, now the app is working fine. But CSS is still broken. I mean it's not what is in the design. Do you want me to implement this design?
Edit: my bad, that's my part to do 0:-)

**Comment by ypahalajani** [comment]
@maaxow : in which design pattern do we maintain this kind of file structure? I was trying to include css files for styling in this project. So, i need to tell the app the location for storing css files. One thing I did, was in public/css folder i mainted a global styles.css and included it in index.html. It worked but to let the app know about all the styling in directives folder(i think, each directive should have it's own styling in it's folder so that the modules become reusable), i'll have to mention the paths for all the css files inside the directives folder. Not sure, how i can do this. For this i need to understand what design patterns/frameworks/tech stack are we using in this project.. This way, I'll get a better understanding of working on this project.

**Comment by maaxow** [comment]
@ypahalajani : i'm not sure i understand all of your comments. But it's a good idea to have a css file for each directives, in their folder.

For information : I use the framework [Angular-Material](https://material.angularjs.org/latest/), with a lot of angular directives, so you can use it. :smiley: 

If you are others questions, don't hesitate. :smiley_cat: 

Thanks again !

**Comment by maaxow** [comment]
@ypahalajani : To load dynamicly css file, we can use the `function loadCss(url)` in require.config.js, and use it as a angular service.

**Comment by ypahalajani** [comment]
@maaxow : template is ready only a colour change is left. I'll work on logic then. Hopefully, this should be done by tomorrow if no big hurdles are met.
![wip_add_money_modal](https://user-images.githubusercontent.com/14838485/31214051-165beb84-a9c7-11e7-8565-bcc46e44089d.png)

**Comment by maaxow** [comment]
@ypahalajani  : It's a very nice work ! :smile: 
Thanks a lot !

**Comment by ypahalajani** [comment]
@maaxow : PR #19 added. Please review :)

**Comment by ypahalajani** [comment]
@maaxow: Hey, This morning, I was looking at the design that you had provided and my design is not close to what was expected. Do you like the current flow or do you want me to make it exactly like the design that you've provided?

Also, my first priority was to complete the modal, so, I had to do inline styling in some places which is bad practice for CSS styling.

**Comment by NitinNair89** [comment]
> Create new modal to add money. Allow multiple adding !
> Like this model :
> ![cw add](https://user-images.githubusercontent.com/6973084/31086446-208820e8-a79a-11e7-8d51-34db46d7f5c9.png)
> When click on "+" => add new row.
> When click on "ADD" => add all rows in mongo.

Rename the "ADD" button to "SAVE". The plus icon at the top-right and the "Add" button at the bottom-right may create confusion for the user. :)

**Comment by ypahalajani** [comment]
@NitinNair89 ok, will send a PR.

**Comment by ypahalajani** [comment]
@maaxow  Could you please assign this to me if this project is still active?

@NitinNair89  Great observation by the way 👍

**Comment by maaxow** [comment]
thanks @NitinNair89 for contributing
This project is not really active, but we can make this active ! 😄
